src/traces/bar/plot.js

```diff
@@ -130,8 +137,7 @@ function plot(gd, plotinfo, cdModule, traceLayer, opts) {
                     mc = cont.color;
                 }
             } else {
-                lw = (di.mlw + 1 || trace.marker.line.width + 1 ||
-                    (di.trace ? di.trace.marker.line.width : 0) + 1) - 1;
```

Comment:
@etpinard is there a need for this second line condition? i.e. <pre>di.trace ? di.trace.marker.line.width : 0</pre>.

Reply:
Wow. Nice catch! That `di.trace` looks pretty odd. Maybe the old [toolpanel](https://github.com/plotly/streambed/tree/master/shelly/toolpanel/static/toolpanel/src/js) uses it. Would you wind trying to `grep` in the streambed directory?

Reply:
![grep](https://user-images.githubusercontent.com/33888540/61416809-02c40280-a8c3-11e9-913a-88443d70f449.png)

Reply:
and what if you grep for `Bar.plot`?

Reply:
![grep_barPlot](https://user-images.githubusercontent.com/33888540/61417056-d957a680-a8c3-11e9-8a1b-a2cb7b7d2a49.png)

Reply:
Great! I think we're good to :hocho: that `di.trace` condition.